src/widgets/index/index.ts

```diff
@@ -111,8 +140,7 @@ function resetPageFromWidgets(widgets: Widget[]): void {
   indexWidgets.forEach(widget => {
     const widgetHelper = widget.getHelper()!;
 
-    // @ts-ignore remove once resetPage is a helper method
-    widgetHelper._change({
+    privateHelperSetState(widgetHelper, {
```

Comment:
If I understand correctly, this doesn't have to be `privateHelperSetState` since we're not providing `_isFromAddWidget` here.

What about just reverting it back to `widgetHelper._change({ ... })`?
If we do that, then we use `privateHelperSetState` only in one place and we could rename it to something like `setStateFromAddWidget` to be more explicit.

What do you think?

Reply:
yes, but that would require us to use _change still, which is a private, so I'd rather use that only in a single function, if that makes sense :)

Reply:
I didn't know _change was used only here. Then it makes sense to me.